wdio.conf.js

```diff
@@ -1,18 +1,18 @@
 const { createCapability } = require('./webdriver-helpers/configuration-helper');
 
 const capabilities = [
-    createCapability("OS X", "Big Sur", "Safari", "14.1"),
-    createCapability("OS X", "Ventura", "Safari", "16.0"),
+    createCapability("OS X", "Monterey", "Safari", "15.6"),
```

Comment:
I was under the impression we wanted to specifically test Safari 14. Please could you confirm we are happy to upgrade beyond it?

Reply:
That is a good point actually. I'll raise with the team.